embedchain/config/InitConfig.py

```diff
@@ -1,12 +1,14 @@
 import os
-
 from embedchain.config.BaseConfig import BaseConfig
+import uuid
```

Comment:
why don't you add imports from the standard library in the first block and then internal imports. thanks.

Reply:
actually it's not standard library, I see you had to add a dependency, but still.

Reply:
Agreed